### PR TITLE
add storage class with retain policy

### PR DESCRIPTION
**Pull request by csaba-fabian**

### CODE REVIEW

File: environments/dev/storage-classes/standard-ssd-lrs-retain.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  labels:
+    addonmanager.kubernetes.io/mode: EnsureExists
+    kubernetes.io/cluster-service: "true"
+  name: standard-ssd-lrs-retain
+parameters:
+  skuname: StandardSSD_LRS
+provisioner: disk.csi.azure.com
+reclaimPolicy: Retain
+volumeBindingMode: WaitForFirstConsumer
+allowVolumeExpansion: true
```
Comment (carslen) [inline environments/dev/storage-classes/standard-ssd-lrs-retain.yaml:13]: Please add blank line at end of file.

**Comment by carslen** [comment]
🔔

**Comment by carslen** [comment]
@csaba-fabian 🛎

**Comment by csaba-fabian** [comment]
opening a new PR

**Pull request closed without merging**